kernel_gateway/services/sessions/handlers.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import notebook.services.sessions.handlers as notebook_handlers
+from ...mixins import TokenAuthorizationMixin, CORSMixin
+
+default_handlers = []
+for path, cls in notebook_handlers.default_handlers:
+    # Everything should have CORS and token auth
+    bases = (TokenAuthorizationMixin, CORSMixin, cls)
```

Comment:
That's probably a good idea since the data in sessions returns kernel ids.  It renders the protection around the kernels endpoint useless.  Maybe one protection setting covers both?